WcaOnRails/app/webpacker/lib/helpers/groups-and-roles-constants.js

```diff
@@ -0,0 +1,12 @@
+export const GROUP_TYPE = {
+  DELEGATE_PROBATION: 'delegate_probation',
+  DELEGATE_REGIONS: 'delegate_regions',
+  TEAMS_COMMITTEES: 'teams_committees',
+  COUNCILS: 'councils',
+};
+
+export const COUNCILS_STATUS = {
+  MEMBER: 'member',
+  SENIOR_MEMBER: 'senior_member',
+  LEADER: 'leader',
+};
```

Comment:
If needed, you can pull this in from Rails. `UserGroup.types` (or is it `UserGroup.group_types`? Depends on how the Rails model `enum` is called) gives you a hash with keys and values. That can be conerted to json using `to_json` convenience method and then loaded in `wca-data.js.erb`.

Sidenote: The wca-data.js.erb file is big, but that's on purpose. Rendering these files (i.e. resolving the Ruby template calls and producing "pure" JS) is very costly and the computation cost inccreases dramatically per file, so we keep all Ruby->JS computations in one single file even if it's not clean design-wise.

Reply:
Okay, done. I've also moved to `wca-data.js.erb`. Everything is good so far.